### ISSUE TITLE
Not able to start FaceLiveness with Validation error

### ISSUE THREAD

**Issue by Izainab22** [aws-amplify/amplify-ui-swift-liveness#73]
Hi,
 
I'm using the same app you've uploaded, and I did all the backend configurations. For creating the sessionId I'm using my own .Net API and I'm calling it in createSession() function. But I'm not able the start the faceLiveness. 

Create Liveness Session Button will call my API and pass the sessionId to FaceLivenessDetectorView, and it's opening the camera but once the recording is started before getting the Oval I'm getting Validation error and the camera still open without doing any thing.

These are the errors am getting in the console: 

![Screenshot 2023-11-09 at 8 44 38 AM](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/61559975/fc5fb4a5-2354-434c-88e3-80d32bd0a7d4)

**Comment by phantumcode** [comment]
can you verify if you have the appropriate IAM policy? you should have permission for each of the following rekognition actions:

auth or unauth role of client/app user to start the liveness session:
"rekognition:StartFaceLivenessSession"

backend service to create and get the liveness results:
"rekognition: CreateFaceLivenessSession"
"rekognition:GetFaceLivenessSessionResults"

Can you provide any sample code or sample app?

**Comment by phantumcode** [comment]
@Izainab22 Can you also provide what device model and iOS version you're seeing this error in?

**Comment by Izainab22** [comment]
I'm able to start the liveness in the demo app you provided now, I was configuring User Pool Group and I didn't give a permission for it. Once I granted the permission it worked.

But I'm using the same code in my app with the same configurations I did in with the demo app, and I'm facing the same issue but the difference is there's no failure and no thing begin printed in the console!

Here is more details: 

I'm using UIKit as a default language in my app. I can open the view and see the camera with the recording and close icons but it is sticking on recording without showing me the oval to detect the face and without any error printed in the console!

These are my CLI configurations:

1. Amplify Init: 

![WfR9K](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/61559975/ea6ba12b-ca4c-48f9-85c4-b79b275c3655)

2. Amplify add auth: 
 
![3kjGm](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/61559975/86af4cad-a60a-4cf4-9fc9-a0451ae51202)

In AWS Console I've granted the below permission to my anauth, auth roles and my user pool group:

{
```
"Effect": "Allow",
"Action": "rekognition:StartFaceLivenessSession",
"Resource": "*"
}
```

I granted only this access because I've developed .Net API's for creating the session and getting the result. And I'm calling these API's in my iOS app.

My code: 

1. My UIKit Controller:

![kwGvQ](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/61559975/fad9cca5-2b56-4bef-bbce-ceaec43fa0b2)

2. My SwiftUI View: 

![JCly1](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/61559975/184fa536-0bd2-46e1-9a89-8738921477d1)

I've been blocked at this stage for almost a week...
Your help will be much much appreciated!
